assets/js/components/channels/ChannelNewCommunityIntro.jsx

```diff
@@ -20,7 +20,54 @@ const ChannelNewCommunityIntro = (props) => {
     <>
       <MobileDisplay>
         <MobileLayout>
-          <div>agsf</div>
+          <div
+            style={{
+              padding: "10px 15px",
```

Comment:
might need a bit more padding here:
<img width="497" alt="image" src="https://user-images.githubusercontent.com/51131939/167760121-fbed4a7e-51cf-4435-9843-c388ae4e2b7b.png">

Reply:
this padding is exactly the same as on other pages